mrp_multi_level/wizards/mrp_multi_level.py

```diff
@@ -814,6 +814,14 @@ def _init_mrp_inventory(self, product_mrp_area):
                 if invs:
                     po.mrp_inventory_id = invs[0]
 
+    def _should_not_build_time_phased_inventory(self, product_mrp_area):
```

Comment:
may be better to understand if we keep these hook methods as positive booleans. For example we already have 'should_create_planned_order', so this one could be 'should_build_time_phased_inventory'

Reply:
Fixed!